.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2023 Robert Bosch GmbH and Microsoft Corporation
```

Comment:
we could merge the tests actions into the ci workflow file

Reply:
Good idea!